src/Type/Php/StrIncrementDecrementFunctionReturnTypeExtension.php

```diff
@@ -0,0 +1,154 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Php;
+
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\FunctionReflection;
+use PHPStan\Reflection\ParametersAcceptorSelector;
+use PHPStan\Type\Constant\ConstantStringType;
+use PHPStan\Type\DynamicFunctionReturnTypeExtension;
+use PHPStan\Type\ErrorType;
+use PHPStan\Type\Type;
+use ValueError;
+use function chr;
+use function count;
+use function function_exists;
+use function implode;
+use function in_array;
+use function is_float;
+use function is_int;
+use function is_numeric;
+use function ord;
+use function preg_match;
+use function str_decrement;
```

Comment:
You need to ignore the errors in a new file referenced from here (for `< 80300`) https://github.com/phpstan/phpstan-src/blob/1.11.x/build/ignore-by-php-version.neon.php